Microsoft.Toolkit.Uwp.SampleApp/SamplePages/FormPanel/FormPanelCode.bind

```diff
@@ -0,0 +1,64 @@
+﻿<Page
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Grid>
+        <ScrollViewer HorizontalScrollBarVisibility="Disabled"
+                      HorizontalScrollMode="Disabled"
+                      VerticalScrollBarVisibility="Auto"
+                      VerticalScrollMode="Auto">
+            <controls:FormPanel HorizontalAlignment="Left"
+                                             VerticalAlignment="Top"
+                                             HorizontalSpacing="@[HorizontalSpacing:Slider:12:0-120]"
+                                             VerticalSpacing="@[VerticalSpacing:Slider:8:0-64]"
+                                             MaxColumnWidth="@[MaxColumnWidth:Slider:240:0-500]"
+                                             Padding="@[Padding:Thickness:0,0,0,0]@">
+                <TextBox Header="Field 1" />
```

Comment:
I think it would be best to generate these in code with random heights and allow the user to add new textboxes. Would be better for testing.